packages/features/bookings/components/BookerSeo.tsx

```diff
@@ -29,7 +41,7 @@ export const BookerSeo = (props: BookerSeoProps) => {
   const title = event?.title ?? "";
   return (
     <HeadSeo
-      title={`${rescheduleUid ? t("reschedule") : ""} ${title} | ${profileName}`}
+      title={`${rescheduleUid && !!bookingData ? t("reschedule") : ""} ${title} | ${profileName}`}
```

Comment:
rely on valid rescheduleUid that should return some bookingData to display title